engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -132,6 +133,16 @@ private ConfigurationImpl() {
 		this.addConstraintDefinitionContributor( defaultConstraintDefinitionContributor );
 	}
 
+	private ValidatedValueUnwrapper<?> createJavaFXUnwrapperClass(TypeResolutionHelper typeResolutionHelper) {
+		try {
+			Class<?> jfxUnwrapperClass = run( LoadClass.action( JFX_UNWRAPPER_CLASS, getClass().getClassLoader() ) );
+			return (ValidatedValueUnwrapper<?>) ( jfxUnwrapperClass.getConstructor( TypeResolutionHelper.class ).newInstance( typeResolutionHelper ) );
```

Comment:
`org.hibernate.validator` itself will never be loaded via the TCCL, since it is in the `org.hibernate.validator` namespace. This is handled in `LoadClass`.